src/main/java/org/broadinstitute/hellbender/bench/TestMath.java

```diff
@@ -0,0 +1,64 @@
+package org.broadinstitute.hellbender.bench;
+
+/**
```

Comment:
Thanks for the example demonstrating.  Lets delete it before the actual merge though